Project.toml

```diff
@@ -6,6 +6,8 @@ version = "4.5.1-DEV"
 Serialization = "9e88b42a-f829-5b0c-bbe9-9e923198166b"
 CSTParser = "00ebfdb7-1f24-5e51-bd34-a7502290713f"
 SymbolServer = "cf896787-08d5-524d-9de7-132aaa0cb996"
+CodeTools = "53a63b46-67e4-5edd-8c66-0af0544a99b9"
```

Comment:
We can't take a dependency on that, it pulls in too many other packages.

Reply:
I can move my PRs to this organization to a different package if you think you can't add a dependency or the code that makes you nervous.

Reply:
This comment was really narrowly talking about CodeTools.jl: I think that would pull at least five new packages in, and because we ship all packages manually for the VS Code extension that would add just too much extra work to keep all of that updated and maintained.

BUT, having said that: I think maybe having a new package that is user facing and exposes a simple API for LanguageServer, SymbolServer, StaticLint etc. might actually be the best design. Such a package could really primarily target REPL users and make things most convenient for them. It could also easily take on any additional dependencies that are not needed for the VS Code extension. Plus, you wouldn't have to deal with the bottleneck of our snail paced review times ;)

The general idea then would be that packages like StaticLint, SymbolServer, LanguageServer etc. are really targeting IDE integration authors, are low level and not user friendly, with a minimal API surface. And (your) new package would expose things for REPL/end users.

@ZacLN and @aminya does that make sense to you? We could actually host such a package here in the `julia-vscode` org, or @aminya you could also host it on your account, I'm indifferent on that.

Reply:
That's OK. I can host a Lint.jl package on my account. 

The old Lint.jl is not registered anymore.

Reply:
Sounds like a good idea! @aminya could you ping me when you create it or if you need any help on it? I'd be lively interested in it. 

@davidanthoff when there's a separate package for standalone linter, maybe this could be mentioned in Readme here as well so it would be easier for people looking for such functionality to find it?

Reply:
Yes, we could definitely change the README here to essentially say "This is a low-level package, go to Lint.jl for something that most users would want to use."